modules/social_features/social_post/src/Plugin/Block/PostGroupBlock.php

```diff
@@ -33,14 +53,20 @@ protected function blockAccess(AccountInterface $account) {
     $group = _social_group_get_current_group();
 
     if (is_object($group)) {
-      if ($group->hasPermission('add post entities in group', $account) && $account->hasPermission("add post entities")) {
+      if (
+        $group->hasPermission('add post entities in group', $account) &&
+        (
+          $account->hasPermission('add post entities') ||
+          $account->hasPermission("add $this->bundle post entities")
```

Comment:
will this work?

`account->hasPermission("add $this->bundle post entities")`

Reply:
Done